.github/workflows/issue-management-orchestration.yml

```
@@ -0,0 +1,368 @@
+        if: always()
+        run: |
+          echo "Labelling Results:"
+          echo "  Labels Applied: ${{ steps.label.outputs.labels_applied }}"
+          echo "  Conflicts: ${{ steps.label.outputs.label_conflicts }}"
+          echo "  Status: ${{ steps.label.outputs.label_status }}"
+
+  enrichment:
+    name: Enrichment Agent
+    runs-on: ubuntu-latest
+    needs: [setup, labeling]
+    if: |
```

actionlint (1.7.12)

[error] if: condition "${{\n  needs.setup.outputs.should_enrich == 'true' &&\n  needs.labeling.outputs.label_status != 'error' &&\n  always()\n}}\n" is always evaluated to true because extra characters are around ${{ }}

(if-cond)